src/layout/flex-grid/src/index.tsx

```diff
@@ -50,14 +12,23 @@ const getVerticalGutterSize = (theme: any): string | number =>
 const getContainerSize = (theme: any): number | number[] =>
   theme?.sizes?.grid?.container?.maxWidth || theme?.grid?.container?.maxWidth
 
-const mediaQueryFunction = (
-  data: number | number[],
-  fn: (colValue: number, index: number) => string
-): string | string[] => {
-  if (Array.isArray(data)) {
-    return data.map(fn)
-  }
-  return fn(data, 0)
+const alwaysGetArray = (valueOrArray: any): any => [].concat(valueOrArray)
```

Comment:
Might be able to use generic here? https://www.typescriptlang.org/docs/handbook/generics.html

`(valueOrArray: T | T[]) T[]`

(untested - don't know if you can use it with `|` like that)

Also, you really missed an opportunity to call this `castArray` 😦 

![](https://images2.minutemediacdn.com/image/upload/c_fill,g_auto,h_1248,w_2220/v1555355765/shape/mentalfloss/castaway_primary.jpg?itok=NzjCI6hA)